libraries/CurieBLE/src/BLEPeripheral.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "BLEPeripheral.h"
 
-static BLEPeripheralEventHandler m_eventHandlers[BLEDeviceLastEvent];
+static BLEPeripheralEventHandler m_eventHandlers[BLEDeviceLastEvent] = {NULL, NULL, NULL, NULL};
```

Comment:
Agree.  The global variable is in _bss segment and it is clear out on boot.  In addition, the initialization is hard code to 4 which may not be true in future.

@sgbihu , please remove the initialization, stay with the previous code for this item.  Please make the correction and make another push ASAP.  Pencil down this Friday.